NetCoreRestApi/BusinessLayer/Manager/StubCategoryModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using DataLayer.Models;
+
+namespace BusinessLayer.Manager
+{
+    public class StubCategoryModel : CategoryModel
```

Comment:
This class is redundant. Please remove it

Reply:
Ok, I will remove it

Reply:
done